Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -250,6 +250,15 @@ open class ComposerVC: _ViewController,
         picker.delegate = self
         return picker
     }()
+    
+    /// The View Controller for taking a picture.
+    open private(set) lazy var cameraVC: UIViewController = {
```

Comment:
nit: In this case, we can actually go without `private(set)` allowing SDK integrators to assign a VC they wanna show

Reply:
With settable `cameraVC`, the customization of a composer from channel/thread VC can look like this: 
```swift
messageComposerVC.cameraVC = MyCameraVC(didSelectPhoto: { [weak messageComposerVC] url in
  try? messageComposerVC?.addAttachmentToContent(from: url, type: .image)
})
```

With `private(set)` the only way to go is:
```swift
class CustomComposerVC: ComposerVC {
    override var cameraVC: UIViewController {
        MyCameraVC(didSelectPhoto: { [weak self] url in
          try? self?.addAttachmentToContent(from: url, type: .image)
        })
    }
}
```